src/toxcore/dht_new/mod.rs

```diff
@@ -27,3 +27,5 @@ pub mod packet;
 pub mod kbucket;
 pub mod packed_node;
 pub mod codec;
+pub mod dht_node;
```

Comment:
Does it have to be named as `dht_node` while the next line is `client`? Shouldn't it be `server`?